src/components/range-facet/bar-chart.js

```diff
@@ -1,49 +1,72 @@
-import React, { useState, useEffect } from "react";
+import React from "react";
+import {Bar} from "react-chartjs-2";
 import {
-  ValueAxis,
-  Chart,
-  BarSeries,
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Tooltip,
+} from 'chart.js';
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  BarElement,
   Tooltip
-} from '@devexpress/dx-react-chart-material-ui';
-import { EventTracker } from '@devexpress/dx-react-chart';
-
```

Comment:
Remove old bar chart library to use `react-chartjs-2`. This is to have more flexible support in hovering, seeing tooltips, etc.